### PR TITLE
New package: radicale2-2.1.10

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by CameronNemo** [comment]
This needs a data migration to work correctly. You could package it as a new package and add a message in radicale 1 on how to migrate.

**Comment by CameronNemo** [comment]
Some notes:

 * Please use https homepages whenever available ("https://radicale.org")
 * Please use SPDX license identifiers whenever possible (in this case "GPL-3.0-or-later")
 * The radicale 1.x package needs a revision bump so that current users see the install message
 * The ignore should be in radicale 1.x package, ignoring the 2.x versions (the 2.x package will already ignore older 1.x versions)
 * The radicale 2.x package needs to conflict with the original radicale package and also provide it (see lxd and lxd-lts packages for an example)
 * The install message could stand to give more guidance. The recommended steps I imagine would be:

   * Disabling the radicale 1.x service
   * (Optionally) backing up the radicale 1.x collections in /var/lib/radicale
   * Moving the old collections to a location that will not conflict with radicale 2.x
   * Exporting the radicale 1.x collections in the 2.x format to /var/lib/radicale
   * Installing radicale 2.x (removing radicale 1.x)
   * Enabling the radicale 2.x service
   * (Optionally) removing the radicale 1.x collections once radicale 2.x service is confirmed to work

**Comment by teldra** [comment]
Thank you for the detailed explanations @CameronNemo. I hope I did it right, this time.

**Comment by CameronNemo** [comment]
Seems good to me, although my father just cut my radicale 1.x collections offline so unfortunately I can't test.

File: srcpkgs/radicale/template [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Template file for 'radicale'
 pkgname=radicale
 version=1.1.6
-revision=1
+revision=2
+noarch=yes
```
Comment (maxice8) [inline srcpkgs/radicale/template]: archs=noarch

File: srcpkgs/radicale2/INSTALL.msg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Disabling the radicale 1.x service
+- (Optionally) backing up the radicale 1.x collections in /var/lib/radicale
+- Moving the old collections to a location that will not conflict with radicale 2.x
+- Exporting the radicale 1.x collections in the 2.x format to /var/lib/radicale
+  (--export-storage path/to/empty/folder)
+- Installing radicale 2.x (removing radicale 1.x)
+- Enabling the radicale 2.x service
+- (Optionally) removing the radicale 1.x collections once radicale 2.x service is confirmed to work
+
+More info at http://radicale.org/1to2/
```
Comment (maxice8) [inline srcpkgs/radicale2/INSTALL.msg]: needs newline here

File: srcpkgs/radicale2/files/radicale/run [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
```
Comment (maxice8) [inline srcpkgs/radicale2/files/radicale/run]: extra line, remove

File: srcpkgs/radicale2/files/radicale/log/run [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
```
Comment (maxice8) [inline srcpkgs/radicale2/files/radicale/log/run]: extra line, remove

File: srcpkgs/radicale2/template [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+revision=1
+noarch=yes
+wrksrc=Radicale-${version}
+build_style=python3-module
+pycompile_module="radicale"
+conf_files="
+/etc/radicale/config
+/etc/sv/radicale/log/run"
+make_dirs="
+/etc/radicale 755 root root
```
Comment (maxice8) [inline srcpkgs/radicale2/template]: 1 whitespace indent

File: srcpkgs/radicale2/template [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'radicale2'
+pkgname=radicale2
+version=2.1.10
+revision=1
+noarch=yes
+wrksrc=Radicale-${version}
```
Comment (maxice8) [inline srcpkgs/radicale2/template]: quote

File: srcpkgs/radicale2/template [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'radicale2'
+pkgname=radicale2
+version=2.1.10
+revision=1
+noarch=yes
```
Comment (maxice8) [inline srcpkgs/radicale2/template]: archs=noarch

File: srcpkgs/radicale2/template [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+/etc/radicale 755 root root
+/usr/share/radicale/ 755 root root
+/var/log/radicale/ 700 root root
+/var/lib/radicale 750 radicale radicale"
+hostmakedepends="python3 python3-setuptools"
+depends="python3-pam python3-requests python3-dulwich python3-vobject
+ python3-dateutil python3-passlib python3-bcrypt"
+# added passlib and bcrypt, since its seems to be the safest option.
+short_desc="Complete calendar and contact storing and manipulating solution"
+maintainer="lemmi <lemmi@nerd2nerd.org>"
```
Comment (maxice8) [inline srcpkgs/radicale2/template]: have yourself as maintainer

**Comment by maxice8** [comment]
@CameronNemo

File: srcpkgs/radicale2/template [hunk 9]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'radicale2'
+pkgname=radicale2
+version=2.1.10
+revision=1
+archs=noarch
+wrksrc="Radicale-${version}"
+build_style=python3-module
+pycompile_module="radicale"
+conf_files="
```
Comment (maxice8) [inline srcpkgs/radicale2/template]: please put conf_files and make_dirs brllow the big variable block  toghether with system_accounts

Comment (teldra) [reply srcpkgs/radicale2/template]: like this?

**Pull request closed without merging**